server/app/routes/review/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+var router = require('express').Router();
+var mongoose = require('mongoose');
+module.exports = router;
+require('../../../db/models');
+var Review = mongoose.model('Review');
+
+
+router.get('/', function (req, res, next) {
+  Review.find({}).exec()
+    .then(function (reviews) {
+      res.status(200).json(reviews);
+    })
+    .then(null, function (err) {
+      console.error("You got an error", err);
+    })
```

Comment:
This `then` block eats any error that falls through, which prevents it from ever reaching the existing error checking middleware. If an error ever comes up while getting all reviews, the user will never receive a response. We should remove this and just let the errors fall through to the error handler naturally